controllers/backplaneconfig_controller.go

```diff
@@ -260,6 +261,10 @@ func (r *MultiClusterEngineReconciler) Reconcile(ctx context.Context, req ctrl.R
 	if err != nil {
 		return ctrl.Result{Requeue: true}, err
 	}
+	result, err = r.removeDeprecatedRBAC(ctx)
+	if err != nil {
+		return result, err
```

Comment:
might be better to intentionally toss the result since we know it isn't used i.e. `_, err = ...` and `return ctrl.Result{}, err`